addons/danger/functions/fnc_tacticsContact.sqf

```diff
@@ -44,10 +46,14 @@ _group setVariable [QGVAR(tacticsTask), "In contact!", EGVAR(main,debug_function
             _group setVariable [QGVAR(isExecutingTactic), nil];
             _group setVariable [QGVAR(tacticsTask), nil];
             _group enableAttack _enableAttack;
+            private _leader = leader _group;
+            if (_leader call EFUNC(main,isAlive) && {_leader call FUNC(isLeader)}) then {
```

Comment:
as far as i can see isLeader already Checks if _leader is alive, so the first call would be redundant.

Reply:
Good catch! Fixed.